gradle.properties

```diff
@@ -25,3 +25,6 @@ defaultPerformanceBaselines=8.11-commit-d7c22864ee3
 
 # Skip dependency analysis for tests
 systemProp.dependency.analysis.test.analysis=false
+
+# Temporary internal option for enabling parallel CC store (default is true)
```

Comment:
We don't document the option for parallel loading as we don't intend anyone other than ourselves using it